### DISCUSSION TITLE
Importing nix-universal-prefetch into nix-community?

### DISCUSSION THREAD

**Discussion by tomfitzhenry** [nix-community/infra#1321] [Ideas]
nix-universal-prefetch is a tool used by a few update scripts in nixpkgs. nix-universal-prefetch was recently removed in https://github.com/NixOS/nixpkgs/pull/321436 , since its upstream repo has been removed from Github. This broke a few update scripts:
* https://github.com/NixOS/nixpkgs/issues/321499
* https://github.com/NixOS/nixpkgs/issues/321491
* https://github.com/NixOS/nixpkgs/issues/321490

The source of this repo still exists: https://github.com/Marcus-Arcadius/nix-universal-prefetch .

To fix this update scripts, should we import https://github.com/Marcus-Arcadius/nix-universal-prefetch into nix-community, and re-package this for nixpkgs?

**Comment by Mic92** [comment]
@tomfitzhenry do you want to take over maintenance?

**Reply by JohnRTitor** [reply]
The issue is not with ownership but rather commitment to maintenance, it is used in update script for apps like Electron, which obviously carries weight.

**Reply by tomfitzhenry** [reply]
Practically I'm not in a good spot to do that maintenance, as I have no relationship with discourse/gitlab/electron.

**Comment by tomfitzhenry** [comment]
I see @Mic92 worked on https://github.com/NixOS/nixpkgs/pull/322179 to replace a use of nix-universal-prefetch with nix-update.

This is far more preferable. Let's leave it with the discourse/gitlab/electron maintainers to ensure that migration from unsupported tool to supported tool.

**Comment by Mic92** [comment]
We can also use https://github.com/nix-community/nurl as an alternative.
